tests/sys/psa_crypto_mac/test_hmac_sha512.c

```diff
@@ -111,11 +100,9 @@ static void test_hashes_hmac_sha512_hash_sequence(void)
 {
     unsigned char key[64];
 
-    /* prepare an empty key */
```

Comment:
hum, you could have left the comments in, they don't do any harm, but can help the reader to follow your test code